admin/controllers/downloadFile.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+module.exports = async (req, res) => {
+  res.setHeader('Content-disposition', 'filename=multiple_pupils_errors.csv')
```

Comment:
should this be here?

Reply:
The filename should be lowercase with hyphens: download-file.js

Reply:
My mistake. It's fixed.